perun-openapi/openapi.yml

```diff
@@ -17387,7 +17387,7 @@ paths:
       tags:
         - ConsentsManager
       operationId: evaluateConsentsForService
-      summary: Evaluates consents for all consent hubs with given service.
+      summary: Evaluates consents ONLY for given service on consent hubs containing it.
```

Comment:
This summary sounds to me like we're ignoring other hub's services' attributes when creating the consents, which we already discussed. Maybe we could explain it a bit more in the docs and the summary, what do you think?